api/serverInfo.php

```diff
@@ -0,0 +1,44 @@
+<?php
+header('Content-Type: application/json');
+
+require_once '../lib/config.php';
+
+$content = $_GET['content'];
+
+switch ($content) {
+    case 'nav-remotebuzzerlog':
+        echo file_get_contents($config['foldersAbs']['tmp'] . '/' . $config['remotebuzzer']['logfile']);
+        break;
+
+    case 'nav-synctodrivelog':
+        echo file_get_contents($config['foldersAbs']['tmp'] . '/' . $config['synctodrive']['logfile']);
+        break;
+
+    case 'nav-myconfig':
+        echo dumpfile($config['foldersAbs']['config'] . '/' . 'my.config.inc.php');
```

Comment:
Better use api/config.php to not share private Data?

Reply:
Yes, makes sense. Done.
https://github.com/andi34/photobooth/blob/e6901242e931097ba0ca988130236bd3e0e8bf1a/api/serverInfo.php#L5
https://github.com/andi34/photobooth/blob/e6901242e931097ba0ca988130236bd3e0e8bf1a/api/serverInfo.php#L20